conventions/documentation/quick-starts.md

```diff
@@ -0,0 +1,265 @@
+  * *Enter the Developer perspective: In the main navigation, click the dropdown menu and select **Developer**.*
+  * *Enter the Admin perspective: In the main navigation, click the dropdown menu and select **Admin**.*
+* Use the "Location, action" structure. Tell a user where to go before telling them what to do.
+  * DO: *In the node.js deployment, hover over the icon.*
+  * DON’T: *Hover over the icon in the node.js deployment.*
+* Keep your product terminology capitalization consistent. Refer to the [CCS product documentation guide](https://redhat-documentation.github.io/supplementary-style-guide/) and [Red Hat’s Corporate Style Guide](https://source.redhat.com/groups/public/word-nerds/red_hat_word_nerds_wiki/corporate_style_guide).
+* If you need to specify a menu type or list as a dropdown, write "dropdown” as one word without a hyphen.
+* Clearly distinguish between a user action and additional information on product functionality.   
+  * User action: *Change the time range of the dashboard by clicking the dropdown menu and selecting **Time range**.*
+  * Additional information: *To look at data in a specific time frame, you can change the time range of the dashboard.*
+* [Avoid directional language](https://www.patternfly.org/v4/ux-writing/writing-for-all-audiences), like *In the top-right corner, click the icon*. Directional language becomes outdated every time UI layouts change. Also, a direction for desktop users may not be accurate for users with a different screen size. Instead, identify something using its name.
+  * DO: *In the navigation menu, click **Settings**.*
+  * DON’T: *In the left-hand menu, click **Settings**.*
+* [Don’t identify items by color alone](https://www.patternfly.org/v4/ux-writing/writing-for-all-audiences), like *Click the gray circle*. Color identifiers  aren’t useful for sight-limited users, especially colorblind users. Instead, identify an item using its name or copy (like button copy).
+  * DO: *The *Success* message indicates a connection.*
+  * DON’T: *The message with a green icon indicates a connection.*
+* Use the second-person point of view (“you”) consistently:
+  * DO: *Set up your environment.*
+  * DON’T: *Let’s set up our environment.*
+* Format UI elements accordingly:
+  * **Copy for buttons, dropdowns, tabs, fields, and other UI controls:** Write the copy as it appears in the UI and bold it.
+  * **All other UI elements—including page, window, and panel names:** Write the copy as it appears in the UI and bold it.
+  * **Code or user-entered text:** Use monospaced font
+  * **Hints:** If a hint to a navigation or masthead element is included, style the text as you would a link.
+  * **CLI commands:** Use monospaced font.
+    * In running text, use a bold monospaced font for a command.
+    * If a parameter or option is a variable value, use an italic monospaced font.
+    * Use a bold monospaced font for the parameter and a monospaced font for the option.
+
+### Modals and in-app messaging
```

Comment:
Should we include that "Leave quick start?" modal screenshot so that folks have copy that they can recycle for each confirmation modal? If all quick starts will have that same modal, it might be a good idea to provide that screenshot so that the same copy is used each time.

Reply:
@spadgett Do you know if the "Leave quick start" modal is built into the quick start framework? Wondering if this is already standardized so contributing teams don't have to worry about it, or whether we should include it here for consistency.